internal/repository/master_index_test.go

```diff
@@ -167,6 +168,14 @@ func TestMasterMergeFinalIndexes(t *testing.T) {
 	rtest.Equals(t, []*repository.Index{idx1, idx2}, finalIndexes)
 
 	mIdx.MergeFinalIndexes()
+	allIndexes := mIdx.All()
+	rtest.Equals(t, 1, len(allIndexes))
+
+	count := 0
```

Comment:
Please rename the variable to `blobCount`.

Reply:
is changed